openedx/features/courseware/tests.py

```diff
@@ -0,0 +1,203 @@
+"""Unit tests for courseware"""
+
+from django.urls import reverse
+
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from openedx.features.courseware import constants
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
```

Comment:
There is no import for `openedx/features/courseware/helpers.py` and `openedx/features/courseware/views.py` in this PR. Test cases for all possible scenarios for functions `get_pre_post_assessments_score` and `pre_post_assessments_score_view` are missing. @zeshanarifios **Can you please verify test coverage is above 80 percent.**

Lets please attach test coverage report to this PR